Datacite JSON.js

```diff
@@ -249,7 +258,69 @@ function doImport() {
 			}
 		}
 	}
-	
+	if (data.relatedItems) {
```

Comment:
@AbeJellinek -- I didn't have sample data for everything here, so looking at this section closely with an eye towards potential breaking spots would be super helpful